js/components/input.js

```diff
@@ -47,9 +54,7 @@ Fliplet.FormBuilder.field('input', {
     }
   },
   created: function() {
-    if (!this.valueIsFromProgress) {
```

Comment:
Hi @squallstar I removed this code as per this Tony's [suggestion](https://weboo.atlassian.net/browse/XOR-629?focusedCommentId=38580) , because of a lot of the complexities come from the fact that we’re choosing to only generate the ID in some scenarios.

Thanks